### ISSUE TITLE
Failures in two L1 TestDQMOnlineClient unit tests

### ISSUE THREAD

**Issue by makortel** [cms-sw/cmssw#32796]
Two unit tests in `DQM/Integration` package are failing since CMSSW_11_3_X_2021-02-02-1100
```
----- Begin Fatal Exception 02-Feb-2021 16:05:10 CET-----------------------
An exception of category 'NoRecord' occurred while
   [0] Processing  stream begin Run run: 334393 stream: 0
   [1] Calling method for module L1TStage2uGT/'l1tStage2uGT'
Exception Message:
No "L1TGlobalPrescalesVetosFractRcd" record found in the EventSetup.n
 Please add an ESSource or ESProducer that delivers such a record.
----- End Fatal Exception -------------------------------------------------
..
---> test TestDQMOnlineClient-l1tstage2_dqm_sourceclient had ERRORS

----- Begin Fatal Exception 02-Feb-2021 16:05:10 CET-----------------------
An exception of category 'NoRecord' occurred while
   [0] Processing  stream begin Run run: 334393 stream: 0
   [1] Calling method for module L1TStage2uGT/'l1tStage2uGtEmul'
Exception Message:
No "L1TGlobalPrescalesVetosFractRcd" record found in the EventSetup.n
 Please add an ESSource or ESProducer that delivers such a record.
----- End Fatal Exception -------------------------------------------------
---> test TestDQMOnlineClient-l1tstage2emulator_dqm_sourceclient had ERRORS
```
http://cmssdt.cern.ch/SDT/cgi-bin/logreader/slc7_amd64_gcc900/CMSSW_11_3_X_2021-02-02-1100/unitTestLogs/DQM/Integration

**Comment by makortel** [comment]
assign dqm,l1

**Comment by makortel** [comment]
Likely to be caused by #32741 (only PR merged into that IB, and seems to be related)

**Comment by christopheralanwest** [comment]
I suspect that the issue is that the global tag at https://github.com/cms-sw/cmssw/blob/dd931b1a61f3948a7c528b815bc15959d56151fa/DQM/Integration/python/config/FrontierCondition_GT_cfi.py#L3 should be replaced with 112X_dataRun3_HLT_v1, which differs from the existing tag by the addition of the new record: 
https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/111X_dataRun3_HLT_v3/112X_dataRun3_HLT_v1

**Comment by jfernan2** [comment]
I have created https://github.com/cms-sw/cmssw/pull/32800 following @christopheralanwest 's comment

**Comment by silviodonato** [comment]
@jfernan2 @christopheralanwest could you clarify if  #32800 #32803 are needed for the MWGR? My understanding was that the global tag is taken elsewhere during the MWGR, but probably I'm wrong.

**Comment by jfernan2** [comment]
At least #32800 is needed since DQM will be running in 112X
Despite GT can be changed manually at P5 DQM machines by DQM DOC, we usually make PRs for every big release change on this file to track and log which GTs are being used for MWGRs, see:

https://github.com/cms-sw/cmssw/commits/18442d459ff6789a404241f5dcd7633ad51a29e7/DQM/Integration/python/config/FrontierCondition_GT_cfi.py
Modification of Express GT is also needed as @mmusich pointed out in 
https://github.com/cms-sw/cmssw/pull/32800#issuecomment-772369949
Thanks

**Comment by silviodonato** [comment]
Solved by  #32800 #32803

**Comment by jfernan2** [comment]
+1
